### ISSUE TITLE
Launching several servers in parallel

### ISSUE THREAD

**Issue by jdat82** [driftyco/ionic-app-scripts#91]
#### Short description of the problem:

I would like to launch the watcher for source code and the web server in parallel. Same applies for others watchers like the one provided by Karma. 
#### What behavior are you expecting?

In my opinion, we should have an easy way to launch the watcher (source code) without having to build. By separating both we could use simple tools like `concurrently` that way for instance:

``` json
scripts:{
  "watch": "ionic-app-scripts build -d && concurrently \"ionic watch\" \"ionic-serve\"" 
}
```

Today if you parallelize `npm run build -- --dev` and `ionic serve` for instance you will open the browser too soon before your bundle is generated.

**Steps to reproduce:**
1. execute a command that launch in parallel the watcher and the web server via any tool you like
2. for instance concurrently which is very handy 

**Which @ionic/app-scripts version are you using?**
0.0.28

**Comment by danbucholtz** [comment]
Hi, sorry, this is out of scope of Ionic at this time.

Thanks,
Dan
